Source/Common/JSONPool.h

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+
+#include <FEXCore/fextl/memory.h>
+#include <FEXCore/fextl/list.h>
+
+#include <tiny-json.h>
+
+namespace FEX::JSON {
+struct JsonAllocator {
+  jsonPool_t PoolObject;
+  fextl::unique_ptr<fextl::list<json_t>> json_objects;
```

Comment:
Unrelated to this PR, but why does this wrap `fextl::list` in a `unique_ptr`?

Reply:
Removed the unique_ptr